CRM/Activity/Form/Activity.php

```diff
@@ -499,10 +499,14 @@ public function preProcess() {
         $params = array('id' => $this->_activityId);
         CRM_Activity_BAO_Activity::retrieve($params, $this->_values);
       }
+
       $this->set('values', $this->_values);
     }
 
     if ($this->_action & CRM_Core_Action::UPDATE) {
+      // We filter out alternatives, in case this is a stored e-mail, before sending to front-end
+      $this->_values['details'] = CRM_Utils_String::stripAlternatives($this->_values['details']);
```

Comment:
This seems like a good idea, but it's a separate topic, right?

Reply:
Hey, @mlutfy, thanks for the feedback. It is kind of a separate topic, but the thing is if we don't do this, the details part of the activity won't be editable, as this is currently done on the template, which breaks the html for the textarea used to show the wysiwyg editor. Let me know if you still think this should better be handled on a different PR/issue.